.cirrus.yml

```
@@ -0,0 +1,68 @@
+linux_aarch64_test_task:
+  compute_engine_instance:
+    image_project: cirrus-images
+    image: family/docker-builder-arm64
+    architecture: arm64
+    platform: linux
+    cpu: 4
+    memory: 16G
+
+  pip_cache:
+    folder: ~/.cache/pip
+
+  setup_script: |
+    # apt-get install -yy gcc python3.10 python3-dev
+    python3 -m pip install --upgrade pip setuptools
+    python3 -m pip install --upgrade 'contourpy>=1.0.1' cycler fonttools kiwisolver importlib_resources \
+    numpy packaging pillow pyparsing python-dateutil setuptools-scm sphinx
+    python3 -m pip install -r requirements/testing/all.txt
+    python3 -m pip install -v .
```

I just realized you have a regular install here. Running `pytest` from the source directory will import from it, and that would break because the extensions are in the install. You'll need an editable install here for it to work:
```suggestion
    python3 -m pip install -ve .
```